terraform/main.tf

```diff
@@ -77,11 +77,27 @@ module "serverless-user" {
   ]
 }
 
+/*
+ * Manage custom domain name resources (used primarily to ease failovers).
+ */
+module "dns-for-failover" {
+  source = "./modules/serverless-api-dns-for-failover"
```

Comment:
Maybe add the version?

Reply:
Local modules aren't versioned, so that wouldn't be an option here.